Dockerfile

```diff
@@ -1,26 +1,22 @@
 FROM bitnami/minideb:bullseye AS setup
 
-RUN apt-get update && apt-get install -y \
-  curl
-  #libgcc
-  # libstdc++
+RUN install_packages curl ca-certificates
 
-RUN mkdir /tmp/vitalam-node/
 WORKDIR /tmp/vitalam-node/
 
 RUN curl -L https://github.com/gruntwork-io/fetch/releases/download/v0.4.2/fetch_linux_amd64 --output ./fetch && chmod +x ./fetch
 
-RUN ./fetch --repo="https://github.com/digicatapult/vitalam-node" --tag="v2.5.1" --release-asset="vitalam-node-v2.5.1-x86_64-unknown-linux-gnu.tar.gz" ./ \
-  && tar -xzf ./vitalam-node-v2.5.1-x86_64-unknown-linux-gnu.tar.gz 
+ARG VITALAM_VERSION
```

Comment:
I'm not sure if this is a good idea but it might be sensible to provide a default of `latest` here. It'll make it easier for someone cloning the repo to build the docker image. That will require the use of a wildcard for the asset in the fetch call below. What do you think @dblane-digicatapult ?

Reply:
```
 => ERROR [setup 5/5] RUN ./fetch --repo="https://github.com/digicatapult/vitalam-node" --tag="latest" --release-asse  0.9s
------
 > [setup 5/5] RUN ./fetch --repo="https://github.com/digicatapult/vitalam-node" --tag="latest" --release-asset="vitalam-node-latest-x86_64-unknown-linux-gnu.tar.gz" ./   && tar -xzf ./vitalam-node-latest-x86_64-unknown-linux-gnu.tar.gz:
#10 0.865 [fetch] time="2022-01-12T16:13:17Z" level=error msg="Could not find assets matching vitalam-node-latest-x86_64-unknown-linux-gnu.tar.gz in release latest\n"
------
```

Reply:
So you can use a regex for `--release-asset` (https://github.com/gruntwork-io/fetch#general-usage)